### ISSUE TITLE
Auxiliary verbs are handled improperly even though they are in the dictionary

### ISSUE THREAD

**Issue by jrencz** [aotaduy/eslint-plugin-spellcheck#7]
Words like `doesn't` `aren't` are in the dictionary but they are reported as error by the plugin rule. In fact they part before the apostrophe is reported as an error which makes me think it's a problem with a regular expression used to handle words

**Comment by jrencz** [comment]
temporary solution with a lot of false negatives:

```json
"skipIfMatch": [
  "\\w+'t$"
]
```

**Comment by aotaduy** [comment]
Same problem as with ordinals. Thanks for the report!

**Comment by aotaduy** [comment]
Corrected !
